main.tf

```
@@ -163,11 +163,11 @@ module "kms_key" {
 
 module "s3_log_storage_bucket" {
   source  = "cloudposse/s3-log-storage/aws"
-  version = "0.26.0"
+  version = "0.28.0"
```

This upgrade is unsafe (significant risk of data loss). 
```suggestion
  version = "1.0.0"
```